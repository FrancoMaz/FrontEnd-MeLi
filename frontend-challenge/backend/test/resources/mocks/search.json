{
  "available_filters": [
    {
      "id": "category",
      "values": [
        {
          "id": "ML2",
          "title": "Celulares",
          "results": 200
        },
        {
          "id": "MLA412445",
          "title": "Libros",
          "results": 250
        },
        {
          "id": "ML3",
          "title": "Electr√≥nica",
          "results": 5
        }
      ]
    }

  ],
  "results": [
    {
      "id": "ML1",
      "title": "Libro 1",
      "price": 3500,
      "thumbnail": "http://image.jpg",
      "currency_id": "ARS",
      "shipping": {
        "free_shipping": true
      },
      "address": {
        "state_name": "Buenos Aires"
      },
      "condition": "new",
      "sold_quantity": 20,
      "category_id": "MLA412445"
    },
    {
      "id": "ML2",
      "title": "Libro 2",
      "price": 4500,
      "thumbnail": "http://image2.jpg",
      "currency_id": "ARS",
      "shipping": {
        "free_shipping": true
      },
      "address": {
        "state_name": "Mendoza"
      },
      "condition": "new",
      "sold_quantity": 5,
      "category_id": "MLA412445"
    }
  ]
}