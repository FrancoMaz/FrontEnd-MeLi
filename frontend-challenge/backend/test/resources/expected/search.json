{
  "author": {
    "name": "Franco",
    "lastname": "Mazzoni"
  },
  "categories": [
    "Libros, Revistas y Comics",
    "Libros"
  ],
  "items": [
    {
      "id": "ML1",
      "title": "Libro 1",
      "price": {
        "currency": "$",
        "amount": 3500,
        "decimals": 2
      },
      "picture": "http://image.jpg",
      "condition": "new",
      "freeShipping": true,
      "stateName": "Buenos Aires"
    },
    {
      "id": "ML2",
      "title": "Libro 2",
      "price": {
        "currency": "$",
        "amount": 4500,
        "decimals": 2
      },
      "picture": "http://image2.jpg",
      "condition": "new",
      "freeShipping": true,
      "stateName": "Mendoza"
    }
  ]
}